app/views/play/level/tome/SpellView.coffee

```diff
@@ -106,6 +107,21 @@ module.exports = class SpellView extends CocoView
     $(@ace.container).find('.ace_gutter').on 'click', @onGutterClick
     @initAutocomplete aceConfig.liveCompletion ? true
 
+    # Create a Spade to 'dig' into Ace.
+    if Spade
```

Comment:
We are always building Spade now, right? We can get rid of the `if Spade` check. (It would need to be `if Spade?` anyway.)